Anagram/Anagram.cs

```diff
@@ -0,0 +1,86 @@
+﻿
+namespace Anagram
+{
+    public partial class Anagram : Form
+    {
+        private String[] wordList;
+        List<string> matchedWordsListFinal;
+        public Anagram()
+        {
+            InitializeComponent();
+            this.wordList = new String[] { };
+            this.matchedWordsListFinal = new List<string>();
+
+        }
+
+        public int WordLoad()
```

Comment:
what does this method do? try to name the methods like verbs or start with verbs.